docs/tutorial.rst

```diff
@@ -0,0 +1,50 @@
+.. _tutorial:
+
+So, you're about to go to Mun?
+==============================
+
+Kerbalism adds about six new ways to get your crew killed. If this is your first attempt at Mun with Kerbalism, read this tutorial or bring some body bags. You have been warned.
+
+Before you go...
+----------------
+
+Let's assume you've just started a new career game and are about to do your first missions to Mun. You didn't expand your VAB or launch pad, so you're limited to vessels with a maximum of 30 parts and 18 tons. A crewed mission to Mun isn't trivial under these circumstances, and with Kerbalism it's going to be a new challenge, even if you've done many missions to Mun in earlier games.
+
+In order to survive with Kerbalism, Kerbals need enough food, water and oxygen for the duration of the entire mission. Command pods contain enough of these to last for 5 days, which is just enough for a quick trip to Mun, like a flyby or a touch and go landing. Anything beyond that will require additional supplies, so stick to Mun - and Mun only. Do not stay for more than a few hours. And don't even think about Minmus, not yet. Any deviation from that will get your crew killed for a number of reasons, starvation just being one of them.
+
+The most important resource on any crewed vessel, after oxygen, is electricity. Without electricity, your crew will suffocate, freeze or burn to death, whichever kills them first. Not having any food left won't be your number one problem when climatization fails and you're about to burn up in the sun. So, let's take care of that first.
```

Comment:
I had some problem without radiator: condensation until electrocution

Reply:
@SylvainLasnier You need the Humidity controller not radiators

Reply:
That is odd. For condensation you have to run an ECLSS with a humidity controller.

While it is true that a radiator would reduce the temperature, cold air actually can store LESS water, so cooling air down would actually INCREASE condensation. However, I don't think that kerbalism simulates that.

Reply:
Understood. Thanks

Reply:
Your correct, Kerbalisms humidity process does not take into account the habitat temperature.